README.md

```diff
@@ -18,15 +18,19 @@ Currently, we have pre-built binaries for macOS (both Intel `x86_64` and Apple S
 
 `pip install viam-sdk`
 
+If you want to install on Windows, you can install from github directly with `pip`:
```

Comment:
Perhaps distinguish between WSL and pure Windows here -- we are fully supported on WSL I believe, and only gRPC supported on Windows